diff --git a/TigGUI/Images/ControlDialog.py b/TigGUI/Images/ControlDialog.py
index 121072d..0308097 100644
--- a/TigGUI/Images/ControlDialog.py
+++ b/TigGUI/Images/ControlDialog.py
@@ -144,7 +144,7 @@ class ImageControlDialog(QDialog):
         QObject.connect(self._whistzoom_timer, SIGNAL("timeout()"), self._zoomHistogramFinalize)
         # set same size for all buttons and controls
         width = 24
-        for w in self._whistunzoom, self._whistzoomin, self._whistzoomout:
+        for w in (self._whistunzoom, self._whistzoomin, self._whistzoomout):
             w.setMinimumSize(width, width)
             w.setMaximumSize(width, width)
         self._whistzoom.setMinimumSize(80, width)
@@ -781,7 +781,7 @@ class ImageControlDialog(QDialog):
         std = measurements.standard_deviation(subset, labels=mask, index=None if mask is None else False)
         dprint(5, "done")
         text = "  ".join([("%s: " + DataValueFormat) % (name, value) for name, value in
-                          ("min", dmin), ("max", dmax), ("mean", mean), ("std", std)] + ["np: %d" % self._subset.size])
+                          (("min", dmin), ("max", dmax), ("mean", mean), ("std", std))] + ["np: %d" % self._subset.size])
         self._wlab_stats.setText(text)
         self._wmore_stats.hide()
         # update markers
